flight/Modules/Actuator/actuator.c

```diff
@@ -532,10 +426,24 @@ static float scaleChannel(float value, float max, float min, float neutral)
 	return valueScaled;
 }
 
+static float channelFailsafeValue(int idx)
+{
+	switch (GetMixerType(idx)) {
+		case MIXERSETTINGS_MIXER1TYPE_MOTOR:
+			return actuatorSettings.ChannelMin[idx];
+		case MIXERSETTINGS_MIXER1TYPE_SERVO:
+			return actuatorSettings.ChannelNeutral[idx];
+		default:
+			// TODO: is this actually right/safe?
+			return 0;
+	}
+
+}
+
 /**
  * Set actuator output to the neutral values (failsafe)
  */
-static void setFailsafe(const ActuatorSettingsData * actuatorSettings, const MixerSettingsData * mixerSettings)
+static void setFailsafe()
```

Comment:
I don't know how I feel about this kind of change. I like it when a function can be wholly tested for correctness, based on inputted and returned values. I don't think this change makes things worse, because the function already depended on external bits. But I wonder if it's possible to move generally more in that direction instead of doubling down on depending on UAVO get/set operations.